src/test/groovy/org/prebid/server/functional/tests/AuctionSpec.groovy

```diff
@@ -246,4 +249,66 @@ class AuctionSpec extends BaseSpec {
         "invalid-stored-request" | { bidReq, storedReq -> bidReq.ext.prebid.storedRequest = storedReq }
         "invalid-stored-impr"    | { bidReq, storedReq -> bidReq.imp[0].ext.prebid.storedRequest = storedReq }
     }
+
+    def "PBS should generate UUID for BidRequest id and merge StoredRequest when generate-storedrequest-bidrequest-id = true or id = {{UUID}}"() {
```

Comment:
1) Same remars as for AMP tests.
2) Please indicate that this is only applicable for APP.